src/main/java/net/sf/jabref/shared/security/Password.java

```diff
@@ -0,0 +1,59 @@
+package net.sf.jabref.shared.security;
+
+import java.security.GeneralSecurityException;
+import java.util.Base64;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.SecretKeySpec;
+
+import net.sf.jabref.shared.DBMSConnector;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+public class Password {
+
+    private static final Log LOGGER = LogFactory.getLog(DBMSConnector.class);
+
+    private static final String ALGORITHM = "AES";
+    private static final String KEY = "ThisIsA128bitKey";
+
+    private final String phrase;
+
+
+    public Password(String phrase) {
+        this.phrase = phrase;
```

Comment:
Could You suggest a new class name for this case, cause `Password` is not adoptable any more. Sth. which symantically means like `PasswordEncrypterAndDecrypter`.